codec/msgpack/msgpack.go

```diff
@@ -0,0 +1,25 @@
+// Package msgpack contains a codec to encode and decode entities in msgpack format
+package msgpack
+
+import (
+    mp "gopkg.in/vmihailenco/msgpack.v2"
+)
+
+const name = "msgpack"
+
+// Codec that encodes to and decodes from msgpack.
+var Codec = new(msgpackCodec)
+
+type msgpackCodec int
+
+func (m msgpackCodec) Marshal(v interface{}) ([]byte, error) {
+	return mp.Marshal(v)
+}
+
+func (m msgpackCodec) Unmarshal(b []byte, v interface{}) error {
+	return mp.Unmarshal(b, v)
+}
+
+func (m msgpackCodec) Name() string {
```

Comment:
It still says "compact and fast data storage format", which is what I was referring to as "ad line". I may be pedantic about commit messages, but they should be neutral, so the correct one in this case is "Add msgpack codec". Simple and to the point.